src/ssvc/decision_points/exploitation.py

```diff
@@ -13,14 +14,19 @@
     " used in the wild by real attackers; there is credible public reporting.",
 )
 
-POC = SsvcDecisionPointValue(
+POC_1 = SsvcDecisionPointValue(
     name="PoC",
     key="P",
     description="One of the following cases is true: (1) private evidence of exploitation is attested but not shared; "
     "(2) widespread hearsay attests to exploitation; (3) typical public PoC in places such as Metasploit"
     " or ExploitDB; or (4) the vulnerability has a well-known method of exploitation.",
 )
 
+POC_2 = SsvcDecisionPointValue(
+    name="Public PoC",
+    key="P",
+    description="One of the following is true: (1) Typical public PoC exists in sources such as Metasploit or websites like ExploitDB; or (2) the vulnerability has a well-known method of exploitation.",
+)
```

Comment:
This is how to modify an existing decision point value: copy it to a new variable name, then create a new decision point (below) that uses it. Modifying existing decision point values can't happen or else we lose versioning.